ProcessMaker/Http/Controllers/Api/ScriptController.php

```diff
@@ -225,9 +225,17 @@ public function store(Request $request)
     public function update(Script $script, Request $request)
     {
         $request->validate(Script::rules($script));
+        $original_attributes = $script->getAttributes();
 
         $script->fill($request->input());
         $script->saveOrFail();
+        
+        unset(
```

Comment:
We should keep the original values for the created_at field.

Reply:
yeah that makes sense